backend/matchmaking/consumers.py

```
@@ -93,186 +94,194 @@ async def match_player(self):
-                await self.send(json.dumps({
-                    "type": "finding_match",
-                    "match_found": False,
-                    "white_player": None,
-                    "black_player": None
-                }))
 
-            
-            await sleep(0.5)
+                
+                await sleep(0.5)
+        except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)